hcl/eval/eval.go

```diff
@@ -155,7 +154,7 @@ func TokensForExpression(expr hhcl.Expression) (hclwrite.Tokens, error) {
 		exprdata = []byte(fnameBytes[len(injectedTokensPrefix()):])
 	} else {
 		var err error
-		exprdata, err = ioutil.ReadFile(string(fnameBytes))
+		exprdata, err = os.ReadFile(string(fnameBytes))
```

Comment:
hmm yeah got this here as well. They were already in my branch, but yeah better to fix this separately.